src/lib/translators/mongo4.ts

```diff
@@ -13,6 +13,15 @@ export interface MongoStep {
   [propName: string]: any;
 }
 
+/** transform a 'concatenate' step into corresponding mongo steps */
+function transformConcatenate(step: Readonly<ConcatenateStep>): MongoStep {
+  const concatArr: any[] = [{ $convert: { input: $$(step.columns[0]), to: 'string' } }];
+  for (const colname of step.columns.slice(1)) {
+    concatArr.push(step.separator, { $convert: { input: $$(colname), to: 'string' } });
+  }
```

Comment:
It should be possible to avoid having to duplicate this from mongo 3.6 translator, I'll make you a small PR to demonstrate this

Reply:
See https://github.com/CharlesRngrd/weaverbird/pull/3